db/seeds.rb

```diff
@@ -24,7 +24,32 @@
 	description: "my description of Futurama",
 	small_cover_url: "/tmp/futurama.jpg",
 	large_cover_url: "/tmp/futurama.jpg",
-	category_id: 3)
+	category_id: 2)
```

Comment:
You should really seed a user, that will keep you from having to recreate your user every time and also will take care of this issue without having to clear your cookies. That is because a user with id 1 will exist.